CHANGELOG.md

```diff
@@ -6,13 +6,19 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
-## [0.21.1] - 2021-09-09
+## [0.22.0] - 2021-09-17
 ### Security
```

Comment:
Let's leave these under the 0.21.1 release and just add the raw k8s manifest stuff to 0.22.0. That may make it easier to tell what's going on in case we ever have to go back and figure it out. :)

Reply:
Done.